Source/ablastr/warn_manager/WarnManager.cpp

```diff
@@ -93,12 +93,14 @@ void WarnManager::RecordWarning(
         } else if(m_abort_on_warning_threshold == WarnPriority::medium) {
             abort_priority = abl_msg_logger::Priority::medium;
         }
+        std::string abort_msg = "A warning with priority '"
+            + abl_msg_logger::PriorityToString(abort_priority)
+            + "' has been raised";
+        abort_msg += ((m_always_warn_immediately) ? "." : (" :\n[" + topic + "] " + text));
```

Comment:
Thanks for the PR! 

Could you post an example of message with the new implementation, either here or in the PR description?

Is the white space between `raised` and (potentially) `:` intentional or should we remove it?
```suggestion
        std::string abort_msg = "A warning with priority '"
            + abl_msg_logger::PriorityToString(abort_priority)
            + "' has been raised";
        abort_msg += ((m_always_warn_immediately) ? "." : (":\n[" + topic + "] " + text));
```

Reply:
I added an example in the description and removed the white space (which was intentional but is a french thing)

Reply:
Ah, I see! Thank you for the updates.